corehq/apps/users/models.py

```diff
@@ -2683,6 +2683,7 @@ class Invitation(models.Model):
     role = models.CharField(max_length=100, null=True)  # role qualified ID
     program = models.CharField(max_length=126, null=True)   # couch id of a Program
     supply_point = models.CharField(max_length=126, null=True)  # couch id of a Location
+    location_id = models.CharField(max_length=126, null=True)  # couch id of a Location (to replace supply_point)
```

Comment:
nit, locations are no longer in couch, so these are really just location UUIDs or `location_id`s

Reply:
actually with that in mind, what would you think of setting this to a foreignkey reference?

Reply:
I think that makes sense [c3f318f](https://github.com/dimagi/commcare-hq/pull/34359/commits/c3f318f328372e904f4b1db42177309211ceb332)

The main benefit being data integrity/robustness right? The tradeoff being potentially higher overhead when querying?

Reply:
yeah, it'd be helpful for data integrity and modeling.  I think it might have a performance impact at write time, but it shouldn't affect reads unless we actually do a join (which should be faster than the in-memory alternative).

This will actually probably be a requirement for upcoming work to make web users manageable by location-restricted users.  We'll need to display a list of invitations associated with a location that's accessible to the user.